src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+let total = 0;
+
+const calculateTotal = (num) => {
+  total += num;
+};
+
+const populations = document.querySelectorAll('.population');
+
+populations.forEach(el => {
```

Comment:
'el' it's not a correct name, try to provide a more descriptive name, like populations.forEach(population => {